src/store/store.ts

```diff
@@ -25,11 +22,6 @@ export const store = configureStore({
     getDefaultMiddleware().concat([aws.middleware, apes.middleware]),
 });
 
-config.oauth.redirectSignIn =
-  window.location.origin + `${appRoutes.auth_redirector}/`;
```

Comment:
when using legacy config from `aws-pull`, oauth signin gets error when fed with csv redirectURLs, so before being passed to `Amplify.configure(..)` we replaced this csv with a single link. 

I'm not sure if this is still the case with `aws-amplify` v6, but it's good to note that this behavior exist

Reply:
@ap-justin That context helps, thanks. This bit was all removed due errors around missing `config.oauth`. Still trying to figure out a way around that. Revert this group of commits, but builds will fail now due to aforementioned error.